shex/sio-model/patientStatusShape.shex

```diff
@@ -0,0 +1,52 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/>
+PREFIX hl7: <http://purl.bioontology.org/ontology/HL7/>
+
+
+:personShape IRI {
+  a [obo:NCBITaxon_9606];
+  sio:SIO_000228 @:personRoleShape;
```

Comment:
The cardinality here is 1 to 1, which is not consistent with the multi-role in the diagram.

Reply:
Yes, I agree but I am not sure if the diagram is correct ;-) lets discuss this with @markwilkinson